orttraining/orttraining/python/training/onnxblock/__init__.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# __init__.py
+
+from .graph import Graph, TrainingGraph
+from .checkpoint_utils import save_checkpoint
+from . import loss, optim
+
+_producer_name = "onnxblock"
```

Comment:
Did not see anything related "onnxblock" in the code, is that a self-explained name? :)